src/components/SubjectViewer/components/AnnotationsPane/SVGLines.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { array, bool, number } from 'prop-types'
+import indexToColor from 'helpers/indexToColor'
+
+export default function SVGLines({ lines, isExtract, reductionIndex }) {
+  const circleWidth = isExtract ? 4 : 10
+  const dashArray = isExtract ? '4' : '0'
+  const strokeWidth = isExtract ? '0.5' : '3'
+
+  return (
+    <g>
+      {lines.map((line, index) => {
+        const color = indexToColor(reductionIndex % 12)
```

Comment:
Oh wait, I forgot to comment on this. Some thoughts: the modulo `%` action should happen _inside_ the `indexToColor()` function itself. This would be a good idea if, e.g. you later modify `indexToColor()` to  have more/less than 12 colours, so you only need to modify the code in one file - `indexToColor.js`

Reply:
Ah, good point. I'll make that change quickly.